packages/sanity/src/core/studio/components/navbar/search/components/SearchHeader.tsx

```diff
@@ -89,7 +89,13 @@ export const SearchHeader = forwardRef<HTMLInputElement, SearchHeaderProps>(func
         {/* (Fullscreen) Close button */}
         {fullscreen && (
           <Card>
-            <Button aria-label="Close search" icon={ArrowLeftIcon} mode="bleed" onClick={onClose} />
+            <Button
+              aria-label="Close search"
+              icon={ArrowLeftIcon}
+              mode="bleed"
+              onClick={onClose}
+              tooltipProps={{content: 'Close search'}}
```

Comment:
This tooltip currently doesn't render correctly, but may be addressed once recent tooltip changes are merged into Sanity UI `alpha` – something to keep an eye on!